src/services/measure.service.js

```diff
@@ -383,6 +383,25 @@ const careGaps = async (args, { req }) => {
   if (query.program) {
     const progArr = Array.isArray(query.program) ? query.program : [query.program];
     // build query AND-ing all potential program parameters
+
+    let measureQuery = {};
+    if (searchTerm) {
+      let prop = Object.keys(searchTerm)[0];
+
+      // for now assume we only support one of a possible identifier property
+
+      if (Array.isArray(searchTerm[prop])) {
```

Comment:
This only covers the situation where the user passes in an array of measure identifiers though. I think we still need to use the search term even if it is not an array